katana/katana.py

```diff
@@ -14,52 +14,37 @@
 
 import os
 import sys
+import subprocess
+class Katana:
+    def __init__(self):
+        pass
+
+    def runProcess(self, osString):
+        subprocess.Popen([osString], shell=True,
+             stdin=None, stdout=None, stderr=None, close_fds=True)
+
+    def to_string(self, args):
+        args[0] = 'manage.py'
+        if args[1] == 'database':
+            args.remove('database')
+        return 'python ' + ' '.join(args)
+
+    def to_katana_dir(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        os.chdir(dir_path)
```

Comment:
Hi @keenan35i,

Please remove line 34, changing directory is not needed.

Thanks.